__tests__/atomic-swap/tx-proposal-create.test.js

```diff
@@ -265,6 +265,8 @@ describe('create tx-proposal api', () => {
     config.setSwapServiceBaseUrl('http://fake-swap-service');
 
     it('should not interact with the service if the feature flag is disabled', async () => {
+      // Deactivating the global feature flag. All tests from now on will have it available.
+      global.constants.SWAP_SERVICE_FEATURE_TOGGLE = false;
```

Comment:
Added this line to the previous PR test to ensure the constants will be always at the expected state.

Reply:
I didn't understand this

Reply:
The comment was badly written, indeed. It deactivates this feature flag for this tests only.
Reworded it on [19bf791](https://github.com/HathorNetwork/hathor-wallet-headless/pull/277/commits/19bf7914c7512d2e59cdf47ef53a93b90c2eecf9)